tests/plugins/widget/manual/deleteparagraph.html

```diff
@@ -0,0 +1,50 @@
+<div id="editor">
+	<div class="simplebox">
+		<h2 class="simplebox-title">Title</h2>
+
+		<div class="simplebox-content">
+			<p>Content...</p>
+		</div>
+	</div>
+	<p>&nbsp;</p>
+    <pre><code class="language-javascript">//Hello another world!</code></pre>
+</div>
+<script>
+	if ( bender.tools.env.mobile ) {
```

Comment:
I see that removing paragraphs doesn't work on my mobile device - did you perhaps do some digging for the cause?

Reply:
I cannot reproduce the issue: it's working on my mobile (Samsung A40 w/Android 10 and Chrome). BTW, I removed the bender.ignore().